ml/rlvr/fine_tuning_orchestrator.py

```
@@ -0,0 +1,463 @@
+        await self.session.commit()
+
+        return {
+            "provider": "anthropic",
+            "method": "prompt_optimization",
+            "examples_used": len(best_examples),
+            "optimized_prompt": optimized_prompt,
+        }
+
+    async def _fine_tune_local(
+        self,
+        run_id: uuid.UUID,
```

Ruff (0.14.5)

Unused method argument: `run_id`

(ARG002)